tests/test_tokenization_auto.py

```diff
@@ -49,3 +50,9 @@ def test_tokenizer_from_pretrained_identifier(self):
         tokenizer = AutoTokenizer.from_pretrained(SMALL_MODEL_IDENTIFIER)
         self.assertIsInstance(tokenizer, BertTokenizer)
         self.assertEqual(len(tokenizer), 12)
+
+    def test_tokenizer_from_model_type(self):
```

Comment:
Same comment as in `test_configuration_auto` here, we should have checks on the common failure cases of the dict mapping.